airbyte-commons/src/main/resources/log4j2.xml

```diff
@@ -169,14 +181,7 @@
 
     <Loggers>
         <Root level="${log-level}">
-            <AppenderRef ref="Default"/>
-            <AppenderRef ref="SimpleDefault"/>
-            <AppenderRef ref="LogSplit"/>
-            <AppenderRef ref="SimpleLogSplit"/>
-            <AppenderRef ref="LogSplitCloud"/>
-            <AppenderRef ref="SimpleLogSplitCloud"/>
-            <AppenderRef ref="AppLogSplit"/>
-            <AppenderRef ref="AppLogSplitCloud"/>
+            <AppenderRef ref="SecretMaskRewrite"/>
```

Comment:
nit: Should we have some comment here explaining that logs should go in the rewrite block above?

Reply:
Will do